sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -871,6 +871,19 @@ same command, targeting the same device. There is the additional limitation that
 to update an executable graph, every node in the graph must be either a kernel
 command or a host-task.
 
+The only characteristics that can differ between two topologically identical
+graphs during an update are:
+
+* The arguments to kernel nodes may be different between graphs. For example,
+  the graph may capture different values for the USM pointers or accessors used
+  in the graph. It is these kernels arguments in `graph` that constitute the
+  inputs & outputs to update to.
+
+* The code that is executed in host-task nodes may be different between graphs,
+  and the update will reflect the code from `graph` host-tasks nodes. However,
+  any changes to executable code between kernel nodes is not counted as
+  topologically identical, and throws an error.
```

Comment:
Should we specify the error code? `invalid`?

Reply:
I think we already cover this under the exception description for the entry-point a few lines down, but putting it here too is more reader friendly, so appended that information to the end of this sentence.